west.yml

```diff
@@ -82,7 +82,7 @@ manifest:
       revision: 575de9d461aa6f430cf62c58a053675377e700f3
       path: modules/hal/st
     - name: hal_stm32
-      revision: dc83915685a5b32a299388e54e0fd259ded86bc4
+      revision: refs/pull/102/head # TODO: change to the committed hash
```

Comment:
That should be the first commit I guess, to make this patch series bisectable.

Reply:
Yeah totally, waiting for that to get merged to change the hash and move it.